system_server has crashed

```
type: crash
SystemUptimeMs: 23434899
Process: system_server
Build: google/akita/akita:14/AP2A.240705.005/2024071600:user/release-keys
Crash-Handler: com.android.internal.os.RuntimeInit$KillApplicationHandler
Loading-Progress: 1.0
Dropped-Count: 0

java.lang.AssertionError: java.lang.IllegalArgumentException: unregistered listener cannot be flushed
	at android.location.provider.LocationProviderBase$Service.lambda$flush$1(LocationProviderBase.java:349)
	at android.location.provider.LocationProviderBase$Service$$ExternalSyntheticLambda3.run(D8$$SyntheticClass:0)
	at android.os.Handler.handleCallback(Handler.java:959)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loopOnce(Looper.java:232)
	at android.os.Looper.loop(Looper.java:317)
	at com.android.server.SystemServer.run(SystemServer.java:1024)
	at com.android.server.SystemServer.main(SystemServer.java:702)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:580)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:872)
Caused by: java.lang.IllegalArgumentException: unregistered listener cannot be flushed
	at com.android.internal.util.Preconditions.checkArgument(Preconditions.java:60)
	at android.location.LocationManager.requestFlush(LocationManager.java:1718)
	at com.android.location.gnss.GnssOverlayLocationProvider.onFlush(GnssOverlayLocationProvider.java:112)
	at android.location.provider.LocationProviderBase$Service.flush(LocationProviderBase.java:344)
	at com.android.server.location.provider.proxy.ProxyLocationProvider$2.run(ProxyLocationProvider.java:207)
	at com.android.server.servicewatcher.ServiceWatcherImpl$MyServiceConnection.runOnBinder(ServiceWatcherImpl.java:257)
	at com.android.server.servicewatcher.ServiceWatcherImpl.lambda$runOnBinder$0(ServiceWatcherImpl.java:126)
	at com.android.server.servicewatcher.ServiceWatcherImpl.$r8$lambda$GCGDYqQZRK7jCQUNZnY8QjXgR2Y(ServiceWatcherImpl.java:0)
	at com.android.server.servicewatcher.ServiceWatcherImpl$$ExternalSyntheticLambda1.run(R8$$SyntheticClass:0)
	at android.os.Handler.handleCallback(Handler.java:959)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loopOnce(Looper.java:232)
	at android.os.Looper.loop(Looper.java:317)
	at android.os.HandlerThread.run(HandlerThread.java:85)
	at com.android.server.ServiceThread.run(ServiceThread.java:46)
```

--- a/akita/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/akita/overlay/frameworks/base/core/res/res/values/config.xml
@@ -302,12 +302,6 @@
     <!-- MMS user agent profile url -->
     <string name="config_mms_user_agent_profile_url" translatable="false">http://www.gstatic.com/android/sms/GKV4X.xml</string>
 
-    <!-- Allow the GPS_PROVIDER to be replaced by the location provider app at run-time. -->
-    <bool name="config_useGnssHardwareProvider" translatable="false">false</bool>
-    <!-- Package name providing GNSS location support. Used only when
-         config_useGnssHardwareProvider is false. -->
-    <string name="config_gnssLocationProviderPackageName" translatable="false">com.google.android.gms</string>
-
     <string name="config_gnssPsdsType">samsung</string>
 
 </resources>

